docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -146,6 +146,8 @@
       url: /Token
     - title: Tooltip
       url: /Tooltip
+    - title: Tooltip v2
```

Comment:
Ooh! Exciting move to put both in stable! Having both in stable makes me feel nervous but I think it's the right call.

Left some notes on the discussion about status: https://github.com/github/primer/discussions/2754#discussioncomment-8440000